git_machete/client.py

```diff
@@ -1855,61 +1855,70 @@ def check_that_forkpoint_is_ancestor_or_equal_to_tip_of_branch(
                 f"Forkpoint {forkpoint_sha} is not ancestor of or the tip "
                 f"of the {branch} branch.")
 
-    def checkout_github_prs(self, pr_no: int) -> None:
+    def checkout_github_prs(self, pr_no: Optional[List[int]] = None, *, all_opened_prs: bool = False, my_opened_prs: bool = False, opened_by: str = None) -> None:
         org: str
         repo: str
         remote: str
         remote, (org, repo) = self.__derive_remote_and_github_org_and_repo()
         current_user: Optional[str] = git_machete.github.derive_current_user_login()
+        if not current_user and my_opened_prs:
+            raise MacheteException(
+                "Could not determine current user name, please check your token.")
+        all_open_prs: List[GitHubPullRequest] = derive_pull_requests(org, repo)
+        prs_numbers: List[GitHubPullRequest] = self.__get_valid_pull_requests(pr_no,
+                                                                              all_opened_prs_from_github=all_open_prs,
+                                                                              org=org,
+                                                                              repo=repo,
+                                                                              all=all_opened_prs,
```

Comment:
Wow lemme check `--all` on my commercial project... that's gonna be like 20+ open PRs 😅

Reply:
@PawelLipski, you can try now :) Worked for my testing repo with 6 PRs but we'll see on greater volume. Seems like you also have there great base to check `--by=<user>` option.

Reply:
Wow tried `--all` on the commercial repo, went smoothly 🎉 

Other than the remark about checking out a branch at the end unnecessary, I'd also think about reducing the amount of spam printed out (ideally, just a single line per each PR?). This was okay-ish for checking out single PR, but is far too much when PRs are checked out in bulk ☹️

Reply:
I see, can we have 2 mods of printing info:
1. verbose (print everything as before) if only one PR number is provided
2. non verbose, print only: `print(fmt(f"Pull request `#{pr.number}` checked out at local branch `{pr.head}`"))`
WDYT?

Reply:
or always  do non-verbose, so its more elegant, as it always behaves the same way

Reply:
I'd even say... always non-verbose (to be reconsidered if the amount of spam turns out too low... but practice shows that it's almost never the case :trollface:)